### DISCUSSION TITLE
Feature Request: Android app navigation buttons in reading view

### DISCUSSION THREAD

**Discussion by a-lagopus** [standardnotes/forum#2504] [Suggestions]
Hi, the Android app is awesome and I love it.
It would be 100% perfect if there would be navigation buttons when in reading view, to go to the previous/next note. As it is now you have to go back and select a next note, which is a bit fiddly.
Swipe left-right to navigate would be even better.

**Comment by atmoio** [comment]
Do you mean some sort of left and right arrows to switch between notes? What would be the use case for that? Seems rare where you have multiple separate notes that are sequential for whatever reason.

**Comment by a-lagopus** [comment]
I'm usually creating one entry per day, and when I re-read them later, I go back in time record-by-record. However in my mind it's more general, any time I get 10 notes, sequential or not, and I'd want to read them all, it would be good to have a button to go through them in one reading pane, without having to go back and select each one of them.

**Comment by atmoio** [comment]
I'd say this is outside the scope of "standardness", but definitely something worth thinking about. Appreciate your feedback.
